x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/import_view/import_view.js

```diff
@@ -666,13 +666,17 @@ function getDefaultState(state, results) {
 
   const timeFieldName = results.timestamp_field;
 
+  const createIndexPattern =
+    capabilities.savedObjectsManagement.edit === false ? false : state.createIndexPattern;
```

Comment:
While this is working ok, I think there's another feature flag we need to take into consideration. I think the intuitive way to give a user permission to create data views is via the `Data View Management` feature flag (see screenshot). The `Saved Objects Management` would only allow creation on the Kibana API level (the corresponding UI doesn't allow to create a data view). The docs also mention the `all Kibana privileges for the Index Pattern Management feature`, which is also required to follow the `Data View Management` link after import.
I think we could either change this to check the data view feature or check if at least one of them is enabled. It's a bit unfortunate that there are two ways to control index pattern creation.

![image](https://user-images.githubusercontent.com/1945390/140098353-56c0c3c3-dcb6-4f68-9ef1-658853b8d6a8.png)

Reply:
Updated in https://github.com/elastic/kibana/pull/117347/commits/93f540cde3270b255b5db59eef55e78d2b4fddda